apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/AbstractCleanupJob.java

```diff
@@ -27,6 +27,7 @@ public abstract class AbstractCleanupJob implements CleanupJob {
           + "  from ${parentTableName} "
           + "  where last_updated between "
           + "  (select min(last_updated) from ${parentTableName}) and (Now() -Interval '${interval} days') "
+          + "  and api_source not like 'S3%' "
```

Comment:
Did we verify the index is being correctly used here? I think it should be fine since it's a prefix search without any string manipulation and we're not using any locale adjustments, but it would be good to confirm.

Reply:
Yes, you hit the nail on the head; since it's a prefix search, the indexing works fine. I looked at different methods of comparison, and this gives us the lowest query cost.